docs/Actions.md

```diff
@@ -770,12 +770,13 @@ mailgun(
 or
 
 mailgun(
-  mailgun_sandbox_domain: "SANDBOX_DOMAIN",
-  mailgun_sandbox_postmaster: "MY_POSTMASTER",
-  mailgun_apikey: "MY_API_KEY",
+  postmaster: "MY_POSTMASTER",
```

Comment:
Sorry, I can't merge this pull request with the changes in the parameters. Could you change it back to the original variable names?